### DISCUSSION TITLE
Swap screen doesn't work until wallet connected

### DISCUSSION THREAD

**Discussion by wjmelements** [airswap/airswap-web#144] [Feedback]
Can make it work with fallback jsonrpc; otherwise don't show it.

**Comment by gpxl-dev** [comment]
Good point. Re: JSONRPC we can't request an order until we know the sender's address, but could assume mainnet and populate the token list (currently we wait so that we know which chain the user is on).

Alternatively could disable and not show as you suggest - I'll create a ticket for this.

**Reply by wjmelements** [reply]
You can request quotes without requesting an order.
